Start of translation to spanish

RockGoliath opened the pull request:
There's some ambiguious adjectives. Here's some stuff that the translation could be debated of:

- reagents in spanish doesn't sound all too good to me, I said catalyst instead.
- tinkerer I don't know how to translate, google doesn't either. I said controller instead.
- awakened translates to "despertado" but that doesn't mean quite the same. I left it like that for now, but there's definitely a case to be made for something more akin to imbued or empowered.
- diviner I translated as the tool the guy in the article is using to divine the location of water https://en.wikipedia.org/wiki/Dowsing. "Divinador" could be used too but I personally prefer "zahorí".
- slate. I initially thought "tableta" but after looking at some dictionary translations "loja" I think fits better.
- sigil could go in lots of ways really, I chose the equivalent of the english word sign.
- blood letter's backpack. To bloodlet in spanish is "hacer una sangria". I have no clue what to translate blood letter's backpack to though.

RockGoliath committed:
Create es_ES.lang

RockGoliath committed:
Update es_ES.lang

WayofTime commented:
A lot of these assumptions are good. "Sigil" is more of some sort of symbol, as in "I wrote a sigil of power in the ground".

TehNut commented on src/main/resources/assets/bloodmagic/lang/es_ES.lang:
```diff
@@ -0,0 +1,164 @@
+item.BloodMagic.sigil.whirlwind.name=Signo del Remolino
+item.BloodMagic.sigil.enderSeverance.name=Signo de División Ender
+item.BloodMagic.sigil.holding.name=Signo de Almacenamiento
+item.BloodMagic.sigil.holding.display=&r%s: &o&n%s
```
This line seems to be the same as the English translation

RockGoliath replied:
Yeah, I was told not to touch stuff that had weird symbols in it. I should have just deleted it.

TehNut replied:
The single characters prefixed with a % (only s in this case) are for string replacements. The ones prefixed by & are for Minecraft formatting codes for colors and such.

TehNut commented on src/main/resources/assets/bloodmagic/lang/es_ES.lang:
```diff
@@ -0,0 +1,164 @@
+item.BloodMagic.orb.archmage.name=Orbe de Sangre de Archimago
+item.BloodMagic.orb.transcendent.name=Orbe de Sangre Transcendente
+
+item.BloodMagic.reagent.incendium.name=Incendium
```
Since these are Latin names, they shouldn't get translated. They also don't need entribs in here

RockGoliath replied:
I didn't touch any but aether, since that one has a proper translation in spanish (éter). Makes sense though, I'll remove the stuff that is not changed that I left in the pull request by mistake.

RockGoliath committed:
Started with blocks

Missing entries:
-Blood Letter's Pack
-Rudimentary Snare
-Draft of Angelus
-Tiled Wooden Path
-Tiled Stone Path
-Tiled Worn Stone Path
-Tiled Obsidian Path
-Raw Stone Path
-Small Raw Stone Bricks
-Raw Stone Tiles
-Accented Raw Stone Tiles
-Raw Inversion Pillar
-Raw Inversion Pillar Base
-Raw Inversion Pillar Cap
-Raw Demon Eye

I think raw in this context means pure? I'm not quite sure so I didn't change those yet. Tiled refers to tiles, like in a bathroom? I'd like clarification on that too. Blood Letter I haven't found a translation for yet. The act of bloodletting I know, but not the profession. As for draft of Angelus, is draft there a draft of wind or draft as in a rough draft?

Regarding the sigil stuff: the translations I can think of are the equivalent of sigil, symbol and sign. Sigil would obviously be more precise but the word is more old fashioned and out of use. I don't know if you have a particular preference either way but I prefer it as it is right now.

RockGoliath committed:
Continuation of English Translation

Missing entries:
-Blood Letter's Pack
-Rudimentary Snare
-Draft of Angelus
-Tiled Wooden Path
-Tiled Stone Path
-Tiled Worn Stone Path
-Tiled Obsidian Path
-Raw Stone Path
-Small Raw Stone Bricks
-Raw Stone Tiles
-Accented Raw Stone Tiles
-Raw Inversion Pillar
-Raw Inversion Pillar Base
-Raw Inversion Pillar Cap
-Raw Demon Eye
-Polished Raw Stone
-Raw Demon Alloy
-Raw Stone Pillar
-Accented Raw Stone Pillar
-Raw Stone Pillar Cap
-Raw Stone Brick Wall
-Raw Small Stone Brick Wall
-Corrosive Small Stone Brick Wall
-Destructive Small Stone Brick Wall
-Vengeful Small Stone Brick Wall
-Steadfast Small Stone Brick Wall
-Raw Stone Wall
-Raw Stone Stairs
-Stores raw life essence
-Better than a Swiffer®!
-I am very close to being moved by this


I think raw in this context means pure? I'm not quite sure so I didn't change those yet. Tiled refers to tiles, like in a bathroom? I'd like clarification on that too. Blood Letter I haven't found a translation for yet. The act of bloodletting I know, but not the profession. As for draft of Angelus, is draft there a draft of wind or draft as in a rough draft?

New issues arise: not entirely sure what small in "Raw Small Stone Brick Wall" refers to. No clue at all what a swiffer is. Not very sure if there's a way to translate the joke "I am very close to being moved by this". Don't know if I should try to come up with it or to take one of the double meanings (and if so, which one).

Regarding the sigil stuff: the translations I can think of are the equivalent of sigil, symbol and sign. Sigil would obviously be more precise but the word is more old fashioned and out of use. I don't know if you have a particular preference either way but I prefer it as it is right now.

RockGoliath committed:
Merge pull request #1 from RockGoliath/patch-3

Continuation of English Translation

WayofTime commented:
Raw means impure, like uncooked food is raw.
Yes, tiled refers to tiles.
Blood Letter isn't a profession, just describing the fact you are letting blood when you use this.
Draft refers to a drink, like a draft of beer.
The "Small" refers to the size of the stone bricks in the wall.
Swiffer is a type of broom that picks up dust, it's a brand-name thing.
You can take the movement meaning for this joke if it isn't translatable.
Sigil is old-fashioned in english as well, so it may be best to use the old fashioned translation. If that doesn't work, go for symbol.
